src/trg-remote-prefs-dialog.c

```diff
@@ -296,6 +299,60 @@ static GtkWidget *trg_rprefs_alt_speed_spin_new(GList ** wl,
     return w;
 }
 
+static void
+trg_rprefs_alt_days_savefunc(GtkWidget * grid, JsonObject * obj,
+                                gchar * key)
+{
+    guint64 days = 0;
+
+    for(gint i = 0, x = 1; i < 7; i++, x<<=1) {
+        GtkWidget *w = gtk_grid_get_child_at (GTK_GRID(grid), i, 0);
+        if(gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(w)))
+            days += x;
+    }
+
+    json_object_set_int_member(obj, key, days);
+}
+
+static GtkWidget *trg_rprefs_alt_days(GList ** wl,
+                                      JsonObject * Obj,
+                                      const gchar * key,
+                                      GtkWidget *alt_time_check)
+{
+#ifdef ENABLE_NL_LANGINFO
+    nl_item abdays[] = {ABDAY_1, ABDAY_2, ABDAY_3, ABDAY_4, ABDAY_5, ABDAY_6, ABDAY_7};
+#else
+    gchar *abdays[] = {_("Sun"), _("Mon"), _("Tue"), _("Wed"), _("Thu"), _("Fri"), _("Sat")};
+#endif
+    GtkWidget *grid = gtk_grid_new();
+    gtk_grid_set_column_homogeneous (GTK_GRID(grid), TRUE);
+    g_signal_connect(G_OBJECT(alt_time_check), "toggled",
+                     G_CALLBACK(toggle_active_arg_is_sensitive), grid);
+
+    guint64 days = json_object_get_int_member(Obj, key);
+
+    for(gint i = 0, x = 1; i < 7; i++, x<<=1) {
+#ifdef ENABLE_NL_LANGINFO
+        GtkWidget *w = gtk_check_button_new_with_label (nl_langinfo(abdays[i]));
```

Comment:
Hmm, thinking about it it seems that `nl_langinfo` uses the locale encoding but all Gtk text *must* be utf-8. If that is the case you need to [convert it](https://developer.gnome.org/glib/stable/glib-Character-Set-Conversion.html).